documentation/platyps-help.md

```diff
@@ -0,0 +1,227 @@
+
+If a cmdlet is added to your module, then the above command will not work; you must first add the corresponding markdown file by running the following command (also used in the previous section):
+```powershell
+New-MarkdownHelp -OutputFolder $output -Module $module -WithModulePage
+```
+
+However, an easier way to update the markdown files to handle **both** changes to cmdlets and new cmdlets is to use the following command:
+```powershell
+Update-MarkdownHelpModule -Path $path
+```
+where `$path` is the path to the folder containing all of the markdown help.
+
+This is the **recommended** command because it updates all the files in the specified folder based on the cmdlets as loaded into your current session.
+
+### Updating MAML
+
+After you have created or updated your markdown for the cmdlets, you need to update the MAML so it can be used with `Get-Help` in PowerShell. 
+
+Updating MAML can be done with the following command:
+```powershell
+New-ExternalPath -Path $path -OutputPath $outputPath -Force
+```
+where `$path` is the path to the folder containing all of the markdown help, and `$outputPath` is the path to the folder containing the existing MAML which will be replaced.
+
+This command generates the MAML based solely on the markdown files. All the information from the cmdlets (thru reflection) are already incorporated into the markdown thru New-MarkdownHelp and Update-MarkdownHelp (or Update-MarkdownHelpModule). This means that you won't need to build/install/import modules to generate help for them.
+
+### Previewing MAML
+
+After you have created or updated your markdown for the cmdlets, you can preview what the corresponding MAML will look like in PowerShell with the `Get-HelpPreview` cmdlet. This cmdlet will generate the MAML from the markdown files and display it in PowerShell so you can see if the help looks correct.
+
+Running the following command will display the help for **all** of your cmdlets:
+```powershell
+Get-HelpPreview -Path $path
+```
+where $path is the path to the MAML help file you generated.
+
+To get the help for a single cmdlet, run the following command:
+```powershell
+$help = Get-HelpPreview -Path $path
+$help | Where-Object {  $_.Name -eq $cmdlet }
+```
+
+## Getting Started
+
+If your service does not currently have any markdown help, follow the below steps to get started.
+
+### Running the `GenerateHelp.ps1` script
+
+If you want to automate this process, run the `tools\platyPS\GenerateHelp.ps1` script with the following command:
+```powershell
+.\GenerateHelp.ps1 -Service $service -PathToRepo $pathToRepo
+```
+where `$service` is the name of your service (*e.g.,* Profile, Compute, Network, etc.) and `$pathToRepo` is the path to the **azure-powershell** repo locally on your machine.
+
+This script can be used for new services looking to create markdown files, or services looking to update their existing markdown files.
+
+### Steps for creating help
+
```

Comment:
Are these the steps that the GenerateHelp script takes?  If so, state that here.  Also, if that is the case, this can go in the 'how it works' section.